packages/Webkul/Shop/src/Http/Resources/MiniCart.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Webkul\Shop\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class MiniCart extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        // It is just demo content will update the resource
+        return [
+            'id'      => $this->id,
+            'name'    => $this->name,
+            'qty'     => $this->qty,
+            'color'   => $this->color,
+            'size'    => $this->size,
+            'images'  => product_image()->getGalleryImages($this),
```

Comment:
Remove these columns and add the discussed columns,

'items' => CartItemResource::collection(...)

Reply:
fixed